src/seedu/addressbook/Main.java

```diff
@@ -88,6 +89,14 @@ private void runCommandLoopUntilExitCommand() {
             recordResult(result);
             ui.showResultToUser(result);
 
+            // Run "List" Command if previous command was Add or Delete
+            if (command.isAutoListEnabled()){
```

Comment:
Can you think of moving this inside the command that was executed?
Doesn't this seem out of place and out of SLAP?